ngrinder-controller/src/main/java/org/ngrinder/script/handler/GroovyGradleProjectScriptHandler.java

```diff
@@ -92,10 +98,16 @@ public boolean prepareScriptEnv(User user, String path, String fileName, String
 		path = PathUtils.join(path, fileName);
 		// Create Dir entry
 		createBaseDirectory(user, path);
+
 		// Create each template entries
-		createFileEntries(user, path, name, url, scriptContent);
-		if (createLib) {
-			createLibraryDirectory(user, path);
+		try {
+			createFileEntries(user, path, name, url, scriptContent);
+			if (createLib) {
+				createLibraryDirectory(user, path);
+			}
+		} catch (NGrinderRuntimeException e) {
+			getFileEntryRepository().delete(user, newArrayList(path));
```

Comment:
What about using just new ArrayList()?
I think we should avoid google guava's utils as much as possible if the matching feature exist in next JDK version.

Reply:
> Do we really need error message? What about just copying templates if not exists during ngrinder start up time?

Yes, I changed to copy resources if they don't exist only at the start up time.